extensions/amp-analytics/0.1/storage-impl.js

```diff
@@ -301,14 +305,20 @@ export class LocalStorageBinding {
 
   /** @override */
   loadBlob(origin) {
-    return new Promise(resolve => {
+    return new Promise((resolve, reject) => {
+      if (!this.win.localStorage) {
+        reject('No localStorage object found');
```

Comment:
I'm not sure these two rejects are needed. If an error is thrown inside the resolver (and resolve has not been called yet), the promise rejects with that error.